src/CKAN.jl

```diff
@@ -0,0 +1,56 @@
+using JSON
+
+immutable CKAN <: DataRepo
+end
+
+base_url(::CKAN) = "https://datadryad.org/mn/object/http://dx.doi.org/"
+
+function description(repo::CKAN, mainpage)
+    j = JSON.parse(text_only(mainpage.root))
+    desc = j["result"]["notes"]
+    authors = [j["result"]["notes"]]
```

Comment:
Should be:

```
 authors = [j["result"]["author"]]
```

Also CKAN has a maintainer field,
that should probably be included:
`j[`results`][`maintainer`]

It also has fields for the email address for both of them, which might also be good to have.